integration/spark2/src/main/scala/org/apache/carbondata/spark/rdd/CarbonDataRDDFactory.scala

```diff
@@ -957,24 +987,36 @@ object CarbonDataRDDFactory {
   private def loadDataFrame(
       sqlContext: SQLContext,
       dataFrame: Option[DataFrame],
+      scanResultRDD: Option[RDD[InternalRow]],
```

Comment:
since you modify this function, can you add parameter description in function header

Reply:
ok